src/Object/UserAttributes.php

```diff
@@ -67,6 +67,8 @@ class UserAttributes extends BaseObject
 
     public ?string $twitter = null;
 
+    private array $_customAttributes = [];
```

Comment:
I dislike using "$_*" to figure out that the variable is private.  
my2cent

Reply:
I used this notation to distinguish it from the real properties of the UserAttributes object expected by Braze. However, being a private detail, we can choose to rename it in the future as well.